modules/health_quest/app/services/health_quest/appointment_service.rb

```diff
@@ -17,15 +17,25 @@ def get_appointments(start_date, end_date, pagination_params = {})
       end
     end
 
-    def get_appointment_by_id(_id)
-      with_monitoring do
-        response =
-          YAML.load_file(Rails.root.join(*appointment_file)).with_indifferent_access
+    def get_appointment_by_id(id)
+      return mock_appointment if Rails.env.development?
 
-        { data: OpenStruct.new(response[:body][:data]) }
+      with_monitoring do
+        response = perform(:get, "#{get_appointments_base_url}/#{id}", {}, headers, timeout: 55)
+        {
+          data: OpenStruct.new(response.body),
+          meta: pagination({})
```

Comment:
Is there not current pagination support? I see that the pagination method has some defaults currently